test/rules_pygen/rules_generator_test.py

```diff
@@ -0,0 +1,88 @@
+import operator
+import unittest
+import unittest.mock
+
+
+class WhenGeneratingBuildFilesTest(unittest.TestCase):
+    def test_that_wheel_compatibility_is_correct(self):
+        from rules_pygen.rules_generator import _check_compatibility
+
+        self.assertTrue(_check_compatibility('pytest-3.2.3-py2.py3-none-any.whl'))
+        self.assertTrue(_check_compatibility('pytest_hidecaptured-0.1.2-py3-none-any.whl'))
+        self.assertTrue(_check_compatibility('Paste-2.0.3-py34-none-any.whl'))
+        self.assertTrue(_check_compatibility('cryptography-2.3.1-cp34-abi3-macosx_10_6_intel.whl'))
+        self.assertTrue(_check_compatibility('cryptography-2.3.1-cp34-abi3-manylinux1_x86_64.whl'))
+
+        self.assertFalse(_check_compatibility('cryptography-1.6-cp35-cp35m-linux_x86_64.whl'))
+        self.assertFalse(_check_compatibility('cryptography-0.7.1-cp27-none-linux_x86_64.whl'))
+        self.assertFalse(_check_compatibility('tornado-5.1.1-cp27-cp27mu-linux_x86_64.whl'))
+
+    @unittest.mock.patch('rules_pygen.rules_generator._calc_sha256sum')
+    def test_that_wheels_can_be_compared(self, mock_checksum):
```

Comment:
This test doesn't really show that wheels can be compared - just that `WheelInfo().platform` is correct. I'd repurpose the name to reflect that better, and check directly `self.assertEqual(wi1.platform, 'linux_...')`, `self.assertEqual(w2.platform, ...)`